doc/debian/jitsi-meet-turn/turnserver.conf

```diff
@@ -0,0 +1,13 @@
+lt-cred-mech
+use-auth-secret
+keep-address-family
+static-auth-secret=__turnSecret__
+realm=jitsi-meet.example.com
+listening-port=4445
+cert=/etc/jitsi/meet/jitsi-meet.example.com.crt
+pkey=/etc/jitsi/meet/jitsi-meet.example.com.key
+
+no-tcp
+listening-port=443
+tls-listening-port=4444
```

Comment:
why are we listening in 3 ports?

Reply:
So udp goes to 443 directly (stun). 
Coturn itself listens to 4444 and that port is used by nginx (localhost:4444) when multiplexing traffic coming on tcp 443.

Reply:
There are some ports wrong here (and duplicated config listening-port), I'm currently testing clean install and fixing all those.